cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md

```diff
@@ -149,18 +149,18 @@ Before exiting the ROOT session, your histogram canvas should look something lik
 
 #### NanoAOD skimming and analysis with `NanoAOD-tools`
 
-The [NanoAOD-tools](https://github.com/cms-nanoAOD/nanoAOD-tools/) repository provides a toolkit to perform operations on NanoAOD files outside the CMSSW software environment. This program can be used to:
+The [NanoAOD-tools](/record/12507) repository provides a toolkit to perform operations on NanoAOD files outside the CMSSW software environment. This program can be used to:
 
 * skim events that fail the validation
 * skim events that fail an analysis condition based on NanoAOD branch contents
 * drop branches from the NanoAOD file to reduce its size
 * compute many common corrections for simulation
 * compute new observables in user-created modules
 
-Skimming data events that fail the validation is a critical feature, so let's see how to do it by following instructions from the [NanoAOD-tools](https://github.com/cms-nanoAOD/nanoAOD-tools/) page. This example will be performed in the ROOT docker container using files from the [Tau 2016H dataset](/record/30565).
+Skimming data events that fail the validation is a critical feature, so let's see how to do it by following instructions from the [NanoAOD-tools](/record/12507) repository README. This example will be performed in the [ROOT docker container](/docs/cms-guide-docker#nanoaod) using files from the [Tau 2016H dataset](/record/30565). The instructions here download files via `git clone`, but it is also possible to download a `.zip` file of the repository from the [record page](/record/12507)
```

Comment:
Here we need to change the wording as the record does not have README (or link to the repository)

Reply:
Now linking like "NanoAOD-tools [repository](link to github)" and then at the end of the paragraph I've said "zip file of the NanoAOD-tools repository from the [record page]()"